traffic_ops/testing/api/v3/withobjs_test.go

```diff
@@ -78,38 +80,40 @@ type TCObjFuncs struct {
+	CDNFederations:                        {CreateTestCDNFederations, DeleteTestCDNFederations},
+	Coordinates:                           {CreateTestCoordinates, DeleteTestCoordinates},
+	DeliveryServices:                      {CreateTestDeliveryServices, DeleteTestDeliveryServices},
+	DeliveryServicesRegexes:               {CreateTestDeliveryServicesRegexes, DeleteTestDeliveryServicesRegexes},
+	DeliveryServiceRequests:               {CreateTestDeliveryServiceRequests, DeleteTestDeliveryServiceRequests},
+	DeliveryServiceRequestComments:        {CreateTestDeliveryServiceRequestComments, DeleteTestDeliveryServiceRequestComments},
+	DeliveryServicesRequiredCapabilities:  {CreateTestDeliveryServicesRequiredCapabilities, DeleteTestDeliveryServicesRequiredCapabilities},
+	Divisions:                             {CreateTestDivisions, DeleteTestDivisions},
+	FederationUsers:                       {CreateTestFederationUsers, DeleteTestFederationUsers},
+	FederationResolvers:                   {CreateTestFederationResolvers, DeleteTestFederationResolvers},
+	Origins:                               {CreateTestOrigins, DeleteTestOrigins},
+	Parameters:                            {CreateTestParameters, DeleteTestParameters},
+	PhysLocations:                         {CreateTestPhysLocations, DeleteTestPhysLocations},
+	Profiles:                              {CreateTestProfiles, DeleteTestProfiles},
+	ProfileParameters:                     {CreateTestProfileParameters, DeleteTestProfileParameters},
+	Regions:                               {CreateTestRegions, DeleteTestRegions},
+	Roles:                                 {CreateTestRoles, DeleteTestRoles},
+	ServerCapabilities:                    {CreateTestServerCapabilities, DeleteTestServerCapabilities},
+	ServerChecks:                          {CreateTestServerChecks, DeleteTestServerChecks},
+	ServerServerCapabilities:              {CreateTestServerServerCapabilities, DeleteTestServerServerCapabilities},
+	ServerServerCapabilitiesForTopologies: {CreateTestServerServerCapabilities, DeleteTestServerServerCapabilitiesForTopologies},
+	Servers:                               {CreateTestServers, DeleteTestServers},
+	ServiceCategories:                     {CreateTestServiceCategories, DeleteTestServiceCategories},
+	Statuses:                              {CreateTestStatuses, DeleteTestStatuses},
+	StaticDNSEntries:                      {CreateTestStaticDNSEntries, DeleteTestStaticDNSEntries},
+	SteeringTargets:                       {SetupSteeringTargets, DeleteTestSteeringTargets},
+	Tenants:                               {CreateTestTenants, DeleteTestTenants},
+	ServerCheckExtensions:                 {CreateTestServerCheckExtensions, DeleteTestServerCheckExtensions},
+	Topologies:                            {CreateTestTopologies, DeleteTestTopologies},
+	TopologyBasedDeliveryServiceRequiredCapabilities: {CreateTestTopologyBasedDeliveryServicesRequiredCapabilities, DeleteTestDeliveryServicesRequiredCapabilities},
```

Comment:
Nit: If this `TCObj` identifier is 40 characters or less (like `TopologyBasedDeliveryServiceRequiredCaps` or `TopologyBasedDeliverySvcReqdCapabilities`), `go fmt` will align the `TCObjFuncs`. Hard to say if that makes it more or less readable overall, I'll leave that call up to you.

Reply:
While it is deeply regrettable that the columns are no longer fully lined up, I think I prefer reading it unabbreviated (at least when looking at the test file that uses it) since its counterpart (`DeliveryServicesRequiredCapabilities`) is also unabbreviated. When we go full topology, we will be able to remove the `TopologyBased` prefix anyways, and keeping it sticking out like a sore thumb like this is a good reminder to follow through on that.

Reply:
Works for me